platform/darwin/src/MGLShapeCollection.h

```diff
@@ -28,9 +28,6 @@ NS_ASSUME_NONNULL_BEGIN
  convenient to use an `MGLPointCollection`, `MGLMultiPolyline`, or
  `MGLMultiPolygon` object, respectively.
  
- A multipolyline is known as a
- <a href="https://tools.ietf.org/html/rfc7946#section-3.1.8">GeometryCollection</a>
- geometry in GeoJSON.
```

Comment:
This sentence was correct except for the inclusion of “multipolyline” instead of “shape collection”.